stories/module-analytics-setup-with-ga4.stories.js

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
@danielgent we already have a file for Analytics setup stories: `stories/module-analytics-setup.stories.js`. Please, add your story to that file.

Reply:
@eugene-manuilov  I added this a month ago (when I started this 3 hour task :stuck_out_tongue: ) and I followed Asvin's steps in setting up a new storybook file

I think I just added it for the feature flag configuration, if that can be done on the other story then that's fine :+1: